client/my-sites/checkout/checkout/composite-checkout-container.js

```diff
@@ -125,7 +79,10 @@ export function isApplePayAvailable() {
 
 const availablePaymentMethods = [ applePayMethod, stripeMethod, paypalMethod ].filter( Boolean );
 
-export default function CompositeCheckoutContainer() {
+const getCart = ( ...args ) => wpcom.getCart( ...args );
```

Comment:
I know I was the one to add these, but we should check to see if this is required or if we can directly pass `wpcom.getCart`/`wpcom.setCart` to the props in `WPCheckoutWrapper`. If we do need them, as I suspect, we should add a comment above this line to say something like "wpcom functions require binding to the wpcom object to function correctly".

Reply:
I think this is required - passing `wpcom.getCart` and `wpcom.setCart` directly gives `TypeError: "this is undefined"`.